resources/views/inc/sidebar.blade.php

```diff
@@ -63,11 +65,11 @@
                 @if(Auth::user()->isAbleTo('snrStaff'))
                     <a class="nav-link {{ Nav::urlDoesContain('dashboard/admin/feedback') }}" href="/dashboard/admin/feedback">Feedback Management</a>
                 @endif
-                @if(Auth::user()->isAbleTo('email'))
+                @if(Auth::user()->isAbleTo('email') || Auth::user()->hasRole('marketing'))
```

Comment:
I don't think we need to modify this since they're added into the email group

Reply:
Removed